hivemq-edge/src/frontend/src/modules/DomainOntology/hooks/useGetDomainOntology.ts

```diff
@@ -0,0 +1,55 @@
+import { useMemo } from 'react'
+
+import { useListDomainNorthboundMappings } from '@/api/hooks/useDomainModel/useListDomainNorthboundMappings.ts'
+import { useListDomainSouthboundMappings } from '@/api/hooks/useDomainModel/useListDomainSouthboundMappings.ts'
+import { useListDomainTags } from '@/api/hooks/useDomainModel/useListDomainTags.ts'
+import { useListTopicFilters } from '@/api/hooks/useTopicFilters/useListTopicFilters.ts'
+import { useListBridges } from '@/api/hooks/useGetBridges/useListBridges.ts'
+import { BridgeSubscription } from '@/modules/DomainOntology/types.ts'
+
+export const useGetDomainOntology = () => {
+  const northMappings = useListDomainNorthboundMappings()
+  const southMappings = useListDomainSouthboundMappings()
+  const tags = useListDomainTags()
+  const topicFilters = useListTopicFilters()
+  const bridges = useListBridges()
+
+  const isLoading =
+    northMappings.isLoading || southMappings.isLoading || tags.isLoading || bridges.isLoading || topicFilters.isLoading
+
+  const isError =
+    northMappings.isError || southMappings.isError || tags.isError || bridges.isError || topicFilters.isError
+
+  const bridgeSubscriptions = useMemo<BridgeSubscription>(() => {
+    const topics = [] as string[]
+    const topicFilters = [] as string[]
+    const mappings = [] as string[][]
```

Comment:
question: it's kinda weird that they are being casted at the beginning. Maybe worth declaring a type here

```suggestion
    const topics: string[] = []
    const topicFilters: string[] = []
    const mappings: string[][] = []
```

Reply:
Yes, it's a legacy piece of code. Fixed